manifests/crc/default-db-included-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -215,6 +215,7 @@ data:
       "experimentNameFormat" : "%datasource%|%clustername%|%namespace%|%workloadname%(%workloadtype%)|%containername%",
       "bulkapilimit" : 1000,
       "isKafkaEnabled" : "false",
+      "perfProfileVersion" : 2.0,
```

Comment:
Update this as well to perfProfileSupportedVersion

Reply:
Done